Source/WebKit/WebProcess/Extensions/Bindings/Cocoa/JSWebExtensionWrapperCocoa.mm

```diff
@@ -249,6 +249,12 @@ id toNSObject(JSContextRef context, JSValueRef valueRef, Class containingObjects
             continue;
 
         auto *itemValue = toJSValue(context, item);
+        if (valuePolicy == ValuePolicy::StopAtTopLevel) {
+            if (itemValue)
+                [result setObject:itemValue forKey:toNSString(propertyName.get())];
```

Comment:
Is there a reason we don't use subscripting here?

    result[toNSString(propertyName.get())] = itemValue;

Reply:
No good reason. I can change it.